torchbenchmark/models/pplbench_beanmachine/__init__.py

```diff
@@ -0,0 +1,53 @@
+import time
+import random
+import torch
+
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import OTHER
+from typing import Tuple
+
+from pplbench.ppls.beanmachine import robust_regression
+from pplbench.models.robust_regression import RobustRegression
+from pplbench.ppls.beanmachine.inference import MCMC
+
+class Model(BenchmarkModel):
+    task = OTHER.OTHER_TASKS
+    DEFAULT_EVAL_BSIZE = 1
+
+    def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
+        super().__init__(test=test, device=device, jit=jit, batch_size=batch_size, extra_args=extra_args)
+
+        if test == "eval" and device != "cpu":
+            raise NotImplementedError("The eval test only supports CPU.")
+
+        # Instantiate model
+        self.model = RobustRegression()
+
+        # Get data for evaluating model
+        self.train_data, self.test_data = self.model.generate_data(seed=int(time.time()))
+
+        # Create inference object with training data
+        self.infer_obj = MCMC(robust_regression.RobustRegression,
+                    self.train_data.attrs)
+        self.infer_obj.compile()
+
+        self.example_inputs = self.test_data
```

Comment:
Can you also implement the `gen_inputs()` interface [example](https://github.com/pytorch/benchmark/blob/main/torchbenchmark/util/framework/vision/model_factory.py#L46), so that users can call it to return a specified number of batched inputs?

Reply:
This interface is optional, we can leave it for later.